src/parsers/manifest/dash/common/indexes/template.ts

```diff
@@ -551,40 +564,37 @@ export default class TemplateRepresentationIndex implements IRepresentationIndex
     const { duration, timescale, endNumber, startNumber = 1 } = this._index;
 
     if (this._isDynamic) {
-      const lastPos = this._manifestBoundsCalculator.estimateMaximumBound();
-      if (lastPos === undefined) {
-        return undefined;
-      }
-      const agressiveModeOffset = this._aggressiveMode ? (duration / timescale) :
-                                                         0;
-      if (this._scaledRelativePeriodEnd !== undefined &&
+      const liveEdge = this._manifestBoundsCalculator.getEstimatedLiveEdge();
+      if (liveEdge !== undefined &&
+          this._scaledRelativePeriodEnd !== undefined &&
           this._scaledRelativePeriodEnd <
-            (lastPos + agressiveModeOffset - this._periodStart) * this._index.timescale) {
+            liveEdge - this._periodStart * this._index.timescale)
```

Comment:
Did you consider adding parenthesis ?
I don't know you thought on that, it may be less readable with parenthesis, but it would be less error prone when editing such a code in the future

Reply:
Yes you're right, will do

Reply:
Done